### ISSUE TITLE
How to change which item is selected?

### ISSUE THREAD

**Issue by Kirkman** [FezVrasta/dropdown.js#14]
I am using a dropdown together with a clickable map. If someone clicks an item in the map, I want to update the `<select>` list to show that item as being selected.

Doing this doesn't seem to work:

```$('#map-container .sample select').val("3");```

**Comment by jaspersorrio** [comment]
Yes I was wondering if there was a way to programmatically change selected option.

**Comment by FezVrasta** [comment]
You can change the selection in the select element and trigger the change event

**Comment by apauly1** [comment]
I tried this the following but it doesn't work, did I miss something?

```
    $('.select-dropdown').on('change', function() {
            $(".select-dropdown").dropdown();
    });
```
```
    $('.select-dropdown').on('change', function() {
            $(".select-dropdown").dropdown({ "dropdownClass": "my-dropdown"});
    });
```

**Comment by FezVrasta** [comment]
What about 

```
$(".select").value("x").trigger("change");
```

**Comment by nekkon** [comment]
$(".select").val("x").trigger("change"); 

Does not work..

**Comment by FezVrasta** [comment]
It was working, looks like some commit removed the feature..

**Comment by nekkon** [comment]
Had to fix it by myself, please support your plugin.. I'll be using it at my projects.

Fix:
        $select.on("change", function(e) {
          var $this = $(e.target);
          if (!$this.val().length) return;

          if (!multi) {
            var $selected;
            if ($select.find(":selected").length) {
              $selected = $select.find(":selected").last();
            }
            else {
              $selected = $select.find("option, li").first();
              // $selected = $select.find("option").first();
            }
            methods._select($dropdown, $selected);
          } else {
            methods._select($dropdown, $select.find(":selected"));
          }
        });

Add this on line 171

**Comment by nekkon** [comment]
Now $(".select").val("x").trigger("change");  // works..

**Comment by FezVrasta** [comment]
Feel free to send a PR

**Comment by Nicks182** [comment]
Thanks for the fix nekkon.

**Comment by adamnguyenit** [comment]
I got the same issue. In my case, I cannot use .trigger("change") because I want 2 selects are both the same value when each one changed. How should I do?

**Comment by sgtshellback** [comment]
Does this fix address when no item is selected? 

`selectedIndex = -1 `

Doesn't seem to work.

**Comment by colinrlly** [comment]
nekkon's fix still works
